Juego.py

```diff
@@ -1061,7 +1061,7 @@ def __init__(self, text, size, color, positionXY):
 
     def doPaint(self):
         if self.text != "":
-            text = unicode(self.text, "UTF-8")
+            text = str(self.text, "UTF-8")
```

Comment:
And this line?

Reply:
In Python 2, two types could be used to represent strings. One of them `str` was a “byte string” type; it represented a sequence of bytes in some particular text encoding and defaulted to ASCII. The other, `unicode` was (as the name implies) a Unicode string type. 

In Python 3, there is one and only one string type. Its name is `str` and it’s Unicode.

So here the `str()` method returns a string which is considered an informal or nicely printable representation of the given object.